repos/system_upgrade/el7toel8/actors/unsupportedupgradecheck/actor.py

```diff
@@ -0,0 +1,66 @@
+import os
+import json
+
+from leapp.actors import Actor
+from leapp.models import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.utils.audit import get_connection
+from leapp import reporting
```

Comment:
Same here. Wondering why didn't linter catch that

Reply:
Reordered.